ext/reflection/php_reflection.c

```diff
@@ -31,6 +31,8 @@
 
 #include "zend.h"
 #include "zend_API.h"
+#include "zend_language_scanner.h"
+#include "zend_language_scanner_defs.h"
```

Comment:
Apparently I could move the necessary declarations from `zend_language_scanner.h` to `zend_compile.h`, but I am puzzled what to do with `zend_language_scanner_defs.h` since it's a generated file so I am hesitant to move anything from there.

Reply:
Thank you :)

Reply:
(But it means that I completely misunderstood what you asked)